ocpmodels/trainers/base_trainer.py

```diff
@@ -441,20 +433,6 @@ def load_checkpoint(self, checkpoint_path):
             self.ema.load_state_dict(checkpoint["ema"])
         else:
             self.ema = None
-        if "scale_dict" in checkpoint and checkpoint["scale_dict"] is not None:
-            logging.info(
-                "Overwriting scaling factors with those loaded from checkpoint. "
-                "If you're generating predictions with a pretrained checkpoint, this is the correct behavior. "
-                "To disable this, delete `scale_dict` from the checkpoint. "
-            )
-            if isinstance(self.model, OCPDataParallel):
-                self.model.module.load_scales(checkpoint["scale_dict"])
-            elif isinstance(self.model, DistributedDataParallel):
-                self.model.module.module.load_scales(checkpoint["scale_dict"])
-            else:
-                raise NotImplementedError(
-                    f"Loading scaling factors not supported for type {type(self.model)}. "
-                )
```

Comment:
Removing these lines reverts https://github.com/Open-Catalyst-Project/ocp/pull/388.. so the following will silently make incorrect predictions if `scale_file` is not specified in the config:

```
from ocpmodels.common.relaxation.ase_utils import OCPCalculator
calculator = OCPCalculator(checkpoint=checkpoint_path)
```

Let's make sure to update [all of the released checkpoints](https://github.com/Open-Catalyst-Project/ocp/blob/main/MODELS.md) to include the new format scaling factors.

And as a safeguard, when loading from a checkpoint for running evaluations (`validate` or `run-relaxation`) with any model that has scaling factors that are unfitted post checkpoint-loading, probably better to throw errors?